### PR TITLE
File upload material

### PR DESCRIPTION
**Pull request by IsaacLessard**

### CODE REVIEW

**Review by Linesmerrill — CHANGES_REQUESTED** [review]
The mdel-button strikes again

**Comment by IsaacLessard** [comment]
Ready for review

File: src/components/FileUpload.vue [hunk 1]
```diff
@@ -8,25 +8,25 @@
...
           <section class="fileSubmission">
             <label>Files to Submit</label>
-            <input class="mdel-button mdl-button--raised" type="file" @change="fileChange">
+            <input class="mdl-button mdl-button--raised" type="file" @change="fileChange">
           </section>
           <section class="pipelineSelection">
             <input type="radio" value="WGS" v-model="type"> Whole Genome Sequence Pipeline
             <input type="radio" value="TSP" v-model="type"> Targeted Sequence Pipeline
-          </section>
+          </section>FF
```
Comment (Linesmerrill) [inline src/components/FileUpload.vue]: FF? freakin typo

**Review by Linesmerrill — CHANGES_REQUESTED** [review]
Added in 'FF'. FileUpload.vue line 16

File: src/components/FileUpload.vue (hunk 1 above)
Comment (IsaacLessard) [reply src/components/FileUpload.vue]: hahaha, sorry about that.

**Pull request merged**